src/js/components/returns/inbound/AddItemsPage.jsx

```diff
@@ -74,24 +76,24 @@ const FIELDS = {
         flexWidth: '4',
         required: true,
         getDynamicAttr: ({
-          updateRow, rowIndex, values, originId, focusField,
+          updateFormValues, rowIndex, values, originId, focusField,
         }) => ({
           locationId: originId,
           onExactProductSelected: ({ product }) => {
             if (focusField && product) {
               focusField(rowIndex, 'quantity');
```

Comment:
just to understand - you say that your workaround with `setTimeout` has to be there, because in this place we don't have `product` yet?
What actually "updates" the product after clicking enter? Is it `onBlur`?
Those are only clarification questions so that I could understand what exactly is going on

Reply:
Thanks for that explanation. I think it is what I thought, but I'm gonna read it in details tomorrow and give a review. Anyway I rely on you in this feature, so if others feel okay with those changes, go ahead, I just wanted to understand the case more if I ever fall to something similar.